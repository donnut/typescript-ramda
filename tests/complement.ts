import R_complement = require('../ramda/dist/src/complement');

declare const string: string;
declare const string_string_to_boolean: (a: string, b: string) => boolean;

// @dts-jest:pass:snap
R_complement(string_string_to_boolean);
// @dts-jest:pass:snap
R_complement(string_string_to_boolean)(string, string);
