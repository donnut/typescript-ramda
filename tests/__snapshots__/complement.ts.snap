// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R.complement(string_string_to_boolean) 1`] = `"(...args: any[]) => boolean"`;

exports[`R.complement(string_string_to_boolean)(string, string) 1`] = `"boolean"`;
